components/progressTracker/theme.css

```diff
@@ -0,0 +1,167 @@
+@import '@teamleader/ui-colors';
+@import '@teamleader/ui-typography';
+@import '@teamleader/ui-animations';
```

Comment:
Please prefix with a tilde `@import '~@teamleader/ui-animations'`,...

Reply:
If I do this it throws an error from the css-loader saying that those CSS modules cannot be found. I also did not see this being used in any other component as I think they are already being imported as modules?

Reply:
Oh you're right, there's a different postcss config in tl-ui, woops

 :ok: by @InstaK on 18/06/2018 13:51:46